inheritance/src/main/java/inheritance/Review.java

```diff
@@ -0,0 +1,21 @@
+package inheritance;
+
+public class Review extends Restaurant {
+    // variables
+    String bodyText;
+    String author;
+    int reviewStars;

```

Comment:
You also want to include the variables you are using in your toString.
`
    String restaurantName;
    float averageRating;
    String price;
`